backend/routers/userRouter.js

```diff
@@ -7,6 +7,16 @@ import { generateToken, isAdmin, isAuth } from '../utils.js';
 
 const userRouter = express.Router();
 
+userRouter.get(
```

Comment:
when i was using then i am getting below error
{
  "message": "Cast to ObjectId failed for value \"top-sellers\" (type string) at path \"_id\" for model \"User\""
}